packages/server/src/routes/sessions.js

```diff
@@ -16,12 +16,18 @@ const {
 // links. Now, we send login links through a clientside redirect instead so this should not be necessary.
 const MAX_ACCESS_TOKEN_USES = 1;
 
+function validatePostLoginRedirectPath(url) {
+    // TODO implement
+    return url !== null;
```

Comment:
What cases is this validation trying to prevent?  Is the idea to replace this functionality?

```
// Prevent them from accessing an undefined route, causes ugly blank screen failure
```

Reply:
You seem to be commenting on one of my intermediate commits before the function was actually implemented; if you look at the [latest version](https://github.com/usdigitalresponse/usdr-gost/pull/288/files#diff-bd5fd4836f4c77fa6cd87a50e817574b0f7f3166dc12c3f34107db964d918c02) on the main PR page, there are some comments explaining. But basically, now that we are actually serving an HTTP redirect, it's important to not expose an [open redirect](https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html) that could be used for e.g. phishing links.

>Is the idea to replace this functionality?
>```
>// Prevent them from accessing an undefined route, causes ugly blank screen failure
>```

No, I removed that because it was redundant -- we already have a VueRouter config that redirects unknown routes back to the dashboard:
```
{
  path: '*',
  redirect: '/dashboard',
}
```

Reply:
Wow, you responded before I could amend.

Sorry!  I realized my mistake right after submitting.